### ISSUE TITLE
Visual studio build fails with 'undeclared identifier'

### ISSUE THREAD

**Issue by trrahul** [ggerganov/llama.cpp#519]
I am getting the following errors when building from visual studio 2022. Any ideas why?

```
llama.cpp\ggml.c(5838,5): error C2065: 'ne12': undeclared identifier
llama.cpp\ggml.c(5839,5): error C2065: 'ne13': undeclared identifier
llama.cpp\ggml.c(5840,5): error C2065: 'ne2': undeclared identifier
llama.cpp\ggml.c(5840,5): error C2065: 'ne12': undeclared identifier
llama.cpp\ggml.c(5841,5): error C2065: 'ne3': undeclared identifier
llama.cpp\ggml.c(5841,5): error C2065: 'ne13': undeclared identifier
llama.cpp\ggml.c(5844,5): error C2065: 'nb00': undeclared identifier
llama.cpp\ggml.c(5852,5): error C2065: 'ne0': undeclared identifier
llama.cpp\ggml.c(5853,5): error C2065: 'ne1': undeclared identifier
llama.cpp\ggml.c(5854,5): error C2065: 'ne2': undeclared identifier
llama.cpp\ggml.c(5855,5): error C2065: 'ne3': undeclared identifier
```

Generated the solution using these commands.

```
cmake -S . -B build/ -D CMAKE_BUILD_TYPE=Release
cmake --build build/ --config Release
```

**Comment by slaren** [comment]
Some variables that are only used in an ```assert``` were commented out in a recent change, and that may have broken debug builds. Building on release should fix that for now, or just uncomment the lines above the errors.

**Comment by slaren** [comment]
This should be fixed in 939ad2d3a56815f480b6fd5ea432a7ee576a7e6b
